src/libs/antares/optimization-options/include/antares/optimization-options/options.h

```diff
@@ -28,16 +28,24 @@ struct SingleOptimOptions
 {
     std::string solverName = "sirius";
     std::string solverParameters;
+
+    // Reusing basis of first optimization (in case we have 2 weekly
+    // linear optimizations [not MILP])
+    bool solverUsesBasis = false;
```

Comment:
Don't we want this to be the dafault behavior for performances ?

Reply:
Yes, you're right.

But the real default values are given by struct **CmdLineOptimOptions** inside **initializeWith(...)**, so I'm not sure the default values in this struct **SingleOptimOptions** are important or not.